### PR TITLE
[NPM] Print running NPM version in stdout.

### PR DESCRIPTION
**Pull request by JungukCho**
**Reason for Change**:

This PR is to print running NPM version in stdout which can help diagnose errors or know tested version (e.g., which version is used for testing).
In addition, this PR replace log with klog for better stdout format in npm.go and main.go

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by matmerr** [comment]
maybe with the cli tools we should add a `version` command

**Comment by JungukCho** [comment]
> maybe with the cli tools we should add a `version` command

That's a good idea.

File: npm/plugin/main.go [hunk 1]
```diff
@@ -71,17 +73,17 @@ func main() {
 	factor := rand.Float64() + 1
 	resyncPeriod := time.Duration(float64(minResyncPeriod.Nanoseconds()) * factor)
 
-	log.Logf("[INFO] Resync period for NPM pod is set to %d.", int(resyncPeriod/time.Minute))
+	klog.Infof("Resync period for NPM pod is set to %d.", int(resyncPeriod/time.Minute))
 	factory := informers.NewSharedInformerFactory(clientset, resyncPeriod)
 
 	npMgr := npm.NewNetworkPolicyManager(clientset, factory, exec.New(), version)
-	metrics.CreateTelemetryHandle(npMgr.GetAppVersion(), npm.GetAIMetadata())
+	metrics.CreateTelemetryHandle(version, npm.GetAIMetadata())
```
Comment (vakalapa) [inline npm/plugin/main.go:80]: Can you fix this lint please, apart from that the PR looks good.

**Review by vakalapa — APPROVED** [review]
lgtm

File: npm/plugin/main.go (hunk 1 above)
Comment (JungukCho) [reply npm/plugin/main.go:80]: As we agreed, it will come with another PR about metric.

**Pull request merged**